MobileCenterAnalytics/MobileCenterAnalytics/Internals/MSAnalyticsDelegate.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class MSEventLog;
+@class MSPageLog;
+@class MSAnalytics;
+
+@protocol MSAnalyticsDelegate <NSObject>
```

Comment:
The reason of the separate delegate class for Crashes is that we expose the delegate to customers.
I don't think this is the case, you can just implement MSChannelDelegate for Analytics. Maybe it is required because you have other methods per log type. Look at below comment.